### PR TITLE
FEAT - Cleaned up specification markdown

### PR DESCRIPTION
**Pull request by carlwilson**
- removed all markdown document titles;
- removed heading numbers;
- removed hand crafted table of contents;
- joined postface material into single document; and
- added typography for non-headings.

### CODE REVIEW

File: specification/implementation/structure/index.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# 4.	CSIP structure
+# SIP structure
```
Comment (karinbredenberg) [inline specification/implementation/structure/index.md]: SIP?

File: specification/index.md [hunk 2]
```diff
@@ -1,48 +1,42 @@
...
-	- [1.2 The Common Specification for Information Packages and Content Information Type Specifications](#12-the-common-specification-for-information-packages-and-content-information-type-specifications)
-	- [1.3 Common Specification for Information Packages, OAIS Information Packages’ specifications and Content Information Type Specifications](#13-common-specification-for-information-packages-oais-information-packages-specifications-and-content-information-type-specifications)
-	- [1.4. Relation to other documents](#14-relation-to-other-documents)
+- [troduction](#introduction)
+	- [he Common Specification for Information Packages and OAIS](#the-common-specification-for-information-packages-and-oais)
+	- [he Common Specification for Information Packages and Content Information Type Specifications](#the-common-specification-for-information-packages-and-content-information-type-specifications)
+	- [ommon Specification for Information Packages, OAIS Information Packages’ specifications and Content Information Type Specifications](#common-specification-for-information-packages-oais-information-packages-specifications-and-content-information-type-specifications)
```
Comment (karinbredenberg) [inline specification/index.md]: Here i get lost in the change. Is letters missing?

File: specification/index.md [hunk 3]
```diff
@@ -1,48 +1,42 @@
...
-		- [3.1. General principles](#31-general-principles)
-		- [3.2. Identification of the Information Package](#32-identification-of-the-information-package)
-		- [3.3.	Structure of the Information Package](#33-structure-of-the-information-package)
-		- [3.4.	Information Package Metadata](#34-information-package-metadata)
+	- [eed for establishing common ground](#need-for-establishing-common-ground)
+	- [equirements for CSIP Information Packages](#principles-for-interoperable-information-packages)
+		- [General principles](#general-principles)
+		- [Identification of the Information Package](#identification-of-the-information-package)
+		- [Structure of the Information Package](#structure-of-the-information-package)
+		- [Information Package Metadata](#information-package-metadata)
```
Comment (karinbredenberg) [inline specification/index.md]: Do we miss letters?

File: specification/index.md [hunk 4]
```diff
@@ -1,48 +1,42 @@
...
-	    - [5.3.5.	Use of the METS file section (element fileSec)](#535-use-of-the-mets-file-section-element-filesec)
-	    - [5.3.6.	Use of the METS structural map (element structMap)](#536-use-of-the-mets-structural-map-structmap)
-		- [5.4. Use of PREMIS](#54-use-of-premis)
-	- [6. Implementation considerations](#6-implementation-considerations)
-	  - [6.1.	Content Information Type Specifications](#61-content-information-type-specifications)
-	    - [6.1.1.	What is a Content Information Type Specification?](#611-what-is-a-content-information-type-specification)
-	    - [6.1.2.	Maintaining Content Information Type Specifications](#612-maintaining-content-information-type-specifications)
-	  - [6.2. Handling large packages](#62-handling-large-packages)
-	  - [6.3.	Handling descriptive metadata within the Common Specification](#63-handling-descriptive-metadata-within-the-common-specification)
+	- [SIP structure](#csip-structure)
```
Comment (karinbredenberg) [inline specification/index.md]: SIP?

File: specification/index.md [hunk 5]
```diff
@@ -1,48 +1,42 @@
...
+		- [General requirements for the use of metadata](#general-requirements-for-the-use-of-metadata)
+		- [Use of METS](#use-of-mets)
+	    - [	Use of the METS root element (element mets)](#use-of-the-mets-root-element-element-mets)
+	    - [	Use of the METS header (element metsHdr)](#use-of-the-mets-header-element-metshdr)
+	    - [Use of the METS descriptive metadata section (element dmdSec)](#use-of-the-mets-descriptive-metadata-section-element-dmdsec)
+	    - [	Use of the METS administrative metadata section (element amdSec)](#use-of-the-mets-administrative-metadata-section-element-amdsec)
+	    - [	Use of the METS file section (element fileSec)](#use-of-the-mets-file-section-element-filesec)
+	    - [	Use of the METS structural map (element structMap)](#use-of-the-mets-structural-map-structmap)
+		- [Use of PREMIS](#use-of-premis)
+	- [mplementation considerations](#implementation-considerations)
```
Comment (karinbredenberg) [inline specification/index.md]: Are we missing letters?

File: specification/postface/postface.md [hunk 6]
```diff
@@ -1,5 +1,78 @@
...
+| 0.15         | 26.08.2016 | Kuldar Aas                | NAE          | Adding available contributions to individual Sections |
+| 0.16         | 05.12.2016 | Andrew Wilson, Kuldar Aas | UoB, NAE     | Major update. Added section on PREMIS. Revision of tables describing use of METS. General text revisions arising from CS meetings. Updates to requirements. |
+| 0.17         | 16.12.2016        | All                | All          | Final discussions, changes and proofreading before delivering the CS to public comment. |
+| 1.0          | 31.01.2017        | Kuldar Aas         | NAE          | Final small editorial additions |
+| 1.1          | 14.05.2018        | Kuldar Aas (editor), DILCIS Board | NAE | Limited proofreading and updates throughout the document. Updates in terminology. Updates in use of METS, ID and referencing section. Improved (more consistent) examples in METS section. |
+| Draft 2.0    | 28.11.2018        | DILCIS Board, E-ARK4ALL | DILCIS Board | Specification updated and released for open review. |
+| 2.0          | 31.05.2019        | DILCIS Board, E-ARK4ALL | DILCIS Board | Specification updated after open review. |
```
Comment (karinbredenberg) [inline specification/postface/postface.md]: And a new revision note is needed

File: specification/implementation/structure/index.md (hunk 1 above)
Comment (karinbredenberg) [reply specification/implementation/structure/index.md]: Needs change?

Comment (karinbredenberg) [reply specification/implementation/structure/index.md]: Not sure here.

File: specification/index.md (hunk 2 above)
Comment (carlwilson) [reply specification/index.md]: This file's to go so don't worry about it.

File: specification/index.md (hunk 3 above)
Comment (carlwilson) [reply specification/index.md]: This file's to go so don't worry about it.

File: specification/index.md (hunk 5 above)
Comment (carlwilson) [reply specification/index.md]: This file's to go so don't worry about it.

File: specification/implementation/structure/index.md (hunk 1 above)
Comment (carlwilson) [reply specification/implementation/structure/index.md]: Got it :)

**Pull request merged**